Library/Formula/mtr.rb

```diff
@@ -1,8 +1,17 @@
 class Mtr < Formula
   desc "'traceroute' and 'ping' in a single tool"
   homepage "https://www.bitwizard.nl/mtr/"
-  url "ftp://ftp.bitwizard.nl/mtr/mtr-0.86.tar.gz"
-  sha256 "c5d948920b641cc35f8b380fc356ddfe07cce6a9c6474afe242fc58113f28c06"
+  head "https://github.com/traviscross/mtr.git"
+
+  stable do
+    url "https://github.com/traviscross/mtr/archive/v0.86.tar.gz"
+    sha256 "7912f049f9506748913e2866068b7f95b11a4e0a855322120b456c46ac9eb763"
+
+    patch do
```

Comment:
Can you please add a explanation of why the patch is needed in a comment in the formula file. Thanks!